code/modules/mob/living/carbon/human/species_types/dwarves.dm

```diff
@@ -136,57 +138,67 @@ GLOBAL_LIST_INIT(dwarf_last, world.file2list("strings/names/dwarf_last.txt")) //
 			filth_counter += 10 //Dwarves could technically chainstun each other in a vomit tantrum spiral.
 	switch(filth_counter)
 		if(11 to 25)
-			if(prob(25))
-				to_chat(owner, "<span class = 'danger'>Someone should really clean up in here!</span>")
+			if(last_filth_spam + 40 SECONDS < world.time)
+				to_chat(owner, "<span class = 'warning'>Someone should really clean up in here!</span>")
+				last_filth_spam = world.time
 		if(26 to 50)
-			if(prob(30)) //Probability the message appears
+			if(prob(7)) //And then the probability they vomit along with it.
```

Comment:
probably convey that this was originally a 0.2*0.3 chance (though that would be prob(6), hmm)

Reply:
It's not such a big change after all, no?